### ISSUE TITLE
Failure response for Displaylayerprogress

### ISSUE THREAD

**Issue by Icesythe7** [UnchartedBull/OctoDash#1425]
**What doesn't work?**
I keep getting the tittle error every 5 mins or so yet it displays the layers and seems to work flawlessly, however this keeps popping up and being annoying.

Edit: I should add that on rare occasions (once every 3 hours maybe if that) it has an error getting response from the printer with a similar reason.

**What did you already try?**
Googled the issue but can't find an instance where everything seemingly works as intended but get random (fake?) errors.

**General Information:**
 - Hardware rpi 4 8gb
 - OS Info octoprint 32bit latest stable
 - OctoDash Version 2.1.2
 - OctoPrint Version 1.5.3

**Additional context**
![20210213_113757](https://user-images.githubusercontent.com/13612149/107855550-04ccab00-6df1-11eb-9315-4ef98fd7317d.jpg)

**Comment by UnchartedBull** [comment]
Duplicate of #1304.

**Comment by UnchartedBull** [comment]
Whoops, I was a little fast there. If the errors are only occurring occasionally you might want to adjust your poll time in the config.json (maybe something like 3000). Nevertheless this will be fixed with v3.0.0 as there will be no more regular API calls which can produce errors.

**Comment by Icesythe7** [comment]
> Whoops, I was a little fast there. If the errors are only occurring occasionally you might want to adjust your poll time in the config.json (maybe something like 3000). Nevertheless this will be fixed with v3.0.0 as there will be no more regular API calls which can produce errors.

3000 didn't change much, will just wait on the v3 release thanks for the response regardless!
